lib/graphql_includable/resolver.rb

```diff
@@ -19,50 +63,80 @@ def self.find_node_by_return_type(node, desired_return_type)
     # Translate a node's selections into `includes` values
     # Combine and format children values
     # Noop on nodes that don't return AR (so no associations to include)
-    def self.includes_for_node(node)
+    def self.includes_for_node(node, includes_manager)
       return_model = node_return_model(node)
       return [] if return_model.blank?
 
-      includes = []
       children = node.scoped_children[node.return_type.unwrap]
-      nested_includes = {}
 
       children.each_value do |child_node|
-        child_includes = includes_for_child(child_node, return_model)
+        includes_for_child(child_node, return_model, includes_manager)
+      end
+    end
 
-        if child_includes.is_a?(Hash)
-          nested_includes.merge!(child_includes)
-        elsif child_includes.is_a?(Array)
-          includes += child_includes
-        else
-          includes << child_includes
+    def self.includes_from_connection(node, parent_model, associations_from_parent_model, includes_manager)
+      return unless node.return_type.fields['edges'].edge_class <= GraphQLIncludable::Relay::EdgeWithNode  # TODO: Possibly basic support for connections with only nodes
```

Comment:
`includes_from_connection` will only work if the connection has been defined using the `define_connection_with_fetched_edge` helper because it attaches the required metadata. This just bails out if it sees a regularly defined connection, I might be able to make it work for direct `nodes` on normal connections 🤔